framer/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "framer",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
There was one change done in one of the PRs on renaming a prop.

Reply:
No need, we don't release it from this branch.

```suggestion
  "version": "1.0.0",
```